scripts/in_container/run_prepare_airflow_packages.sh

```diff
@@ -55,9 +55,10 @@ function prepare_airflow_packages() {
         else
             if [[ ${AIRFLOW_VERSION} != *${VERSION_SUFFIX_FOR_PYPI} ]]; then
                 echo
-                echo "${COLOR_RED}The requested PyPI suffix ${VERSION_SUFFIX_FOR_PYPI} does not match the one in ${AIRFLOW_VERSION}. Exiting.${COLOR_RESET}"
+                echo "${COLOR_YELLOW}The requested PyPI suffix ${VERSION_SUFFIX_FOR_PYPI} does not match the one in ${AIRFLOW_VERSION}. Overriding it with one from ${AIRFLOW_VERSION}.${COLOR_RESET}"
                 echo
-                exit 1
+                VERSION_SUFFIX_FOR_PYPI=$(printf "%s\n" "${AIRFLOW_VERSION##*.}")
```

Comment:
This is what I was able to come up with
```suggestion
                VERSION_SUFFIX_FOR_PYPI="${AIRFLOW_VERSION/#[0-9].[0-9].[0-9]/}}"
```

Reply:
Fix pushed to do more sanity checks

Reply:
Oh much nicer.